static/app/utils/fields/index.ts

```diff
@@ -20,6 +20,7 @@ export enum FieldKey {
   ASSIGNED_OR_SUGGESTED = 'assigned_or_suggested',
   BOOKMARKS = 'bookmarks',
   BROWSER_NAME = 'browser.name',
+  BROWSER_VERSION = 'browser.version',
```

Comment:
Since `browser.version` is not a field available in Discover yet, we should hold back on adding this now and check with other teams whether we're open to introduce this field across the board.

If we stick with just `browser`, we'll have to find a different workaround to map this to protocol fields.